src/Education/IRuleHelpXamlBuilder.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
```

Comment:
I am thinking writing tests for this doesn't make much sense but please correct me if I am wrong.

Reply:
I agree, it doesn't make any sense yet.